docs/src/howto-guides/017_apama_software_management_plugin.md

```diff
@@ -1,70 +1,40 @@
 # Apama Software Management Plugin
 
-The apama plugin can be used to install and manage apama artefacts.
-This plugin can be used to install an apama project on a device as well as inject monitor files into an already installed project.
+The Apama plugin can be used to install Apama projects using the Cumulocity software management feature.
 
+> Note: This plugin expects an Apama installation on the device.
 
+## Install Apama artifacts from Cumulocity
 
-## Download and Install Apama Plugin
```

Comment:
Why did you remove this section on installing the Apama software management plugin itself? Is the plugin now part of the Apama installation itself? Otherwise, how to install it?

Reply:
The method of installation will change and I do not know what to put here yet. Some details have yet to be worked out, hence this is still a draft pull request.

Reply:
91ba8fb